src/BudgetManager/BudgetManager.Model/Enums/LimitPeriods.cs

```diff
@@ -2,7 +2,7 @@
 {
     public enum LimitPeriods
     {
```

Comment:
need to add NoDate or None to avoid nullable Enums

Reply:
I think it's also a good idea to assign negative values to invalid enum names.